src/BaGet.AWS/Configuration/S3StorageOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+using BaGet.Core.Validation;
+
+namespace BaGet.AWS.Configuration
+{
+    public class S3StorageOptions
+    {
+        [RequiredIf(nameof(KeySecret), null, IsInverted = true)]
+        public string KeyId { get; set; }
```

Comment:
It seems like `AccessKey` and `SecretKey` are the names used by the AWS SDK for `KeyId` and `KeySecret`. Could we use their terminology instead?